packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Build a SCIM provisioning integration overview
+excerpt: Learn about the steps required to create a provisioning integration using SCIM.
```

Comment:
How about changing this so that the content is mentioned first (similar comments on "Learn about..."... it's a guide so that's implicit... well, if it's a good guide anyway and ours are ;-)):
```
Create a provisioning integration with SCIM in Okta.
```

Reply:
done